src/applications/appeals/995/utils/focus.js

```diff
@@ -5,15 +5,20 @@ import {
 } from 'platform/utilities/ui';
 import { $ } from 'platform/forms-system/src/js/utilities/ui';
 
+let timer;
 export const focusEvidence = (_index, root) => {
-  setTimeout(() => {
+  clearTimeout(timer);
+  timer = setTimeout(() => {
     const error = $('[error]', root);
-    if (error) {
-      scrollToFirstError();
-      focusElement(error);
-    } else {
-      scrollTo('topContentElement');
-      focusElement('#main h3', null, root);
+    timer = null;
```

Comment:
Should this be a call to `clearTimeout(timer)` instead? You call `clearTimeout(timer)` before the call to `setTimeout()` but the timeout function calls itself. It results in a weird loop where you have:
- call to `focusEvidence()`
- call to `clearTimeout(timer())`
- call to `timer = setTimeout()`
- assignment of `timer = null`
- call to `focusEvidence()`

That `timer = null` call seems out of place.

Reply:
Removed!